spec/traits.dd

```diff
@@ -580,6 +581,17 @@ $(H3 $(GNAME classInstanceSize))
         polymorphic type.
         )
 
+$(H3 $(GNAME classInstanceAlignment))
+
+        $(P Takes a single argument, which must evaluate to either
+        a class type or an expression of class type.
+        The result
+        is of type $(CODE size_t), and the value is the alignment
+        of a runtime instance of the class type.
+        It is based on the static type of a class, not the
+        polymorphic type.
+        )
```

Comment:
I would suggest 80 char text wrapping.

```suggestion
        $(P Takes a single argument, which must evaluate to either a class type
        or an expression of class type.
        The result is of type $(CODE size_t), and the value is the alignment of
        a runtime instance of the class type. It is based on the static type of
        a class, not the polymorphic type.
        )
```

Reply:
[The block is a copy-pasta from classInstanceSize above.]

Reply:
Right, we can refactor those in a subsequent PR.